tests/e2e/cypress/cypress/support/commands.js

```diff
@@ -223,3 +223,21 @@ Cypress.Commands.add("checkDocumentEssentialFields", (document) => {
   cy.get('#doc-provision-activity-0').should('contain', document.provisionActivity.statement.place + ' : ' + document.provisionActivity.statement.agent + ', ' + document.provisionActivity.statement.date);
 });
 //#endregion ------------------------------------------------------------------>
+
+//#region Record
+Cypress.Commands.add("populateOrderLine", (order_number, acq_account_reference, document_reference, order_line_status, amount, note) => {
+
+  cy.get('.mt-4 > .btn').click({multiple : true})
```

Comment:
selector....

Reply:
I agree with you. But this need supplementary changes in `rero-ils-ui`, sometimes in `ng-core` to adapt UI and get more relevant selectors.
This could be done in a later stage in my opinion.